nibabel/analyze.py

```diff
@@ -699,7 +699,7 @@ def set_zooms(self, zooms):
         ndim = dims[0]
         zooms = np.asarray(zooms)
         if len(zooms) != ndim:
-            raise HeaderDataError('Expecting %d zoom values for ndim %d' % (ndim, ndim))
```

Comment:
Just wondering, does `ndim` really must appear twice in the error message?

Reply:
I think it's reasonable to say you were expecting N zooms for an N-dim image. I think it would also make sense to rewrite something along the lines of `"Zooms must have one value per dimension (ndim={ndim}). Received: {zooms}."`